container/docker/factory.go

```diff
@@ -62,6 +63,10 @@ func UseSystemd() bool {
 	return useSystemd
 }
 
+func RootDir() string {
+	return *dockerRootDir
```

Comment:
We can also look at the Docker config file. Customization is usually done there.